packages/components/data-table/src/cell.js

```diff
@@ -97,6 +113,15 @@ DataCell.propTypes = {
   isCondensed: PropTypes.bool,
   isTruncated: PropTypes.bool,
   shouldIgnoreRowClick: PropTypes.bool,
+  shouldRenderCollapseButton: requiredIf(
+    PropTypes.bool,
+    (props) => props.isTruncated
+  ),
```

Comment:
```suggestion
  shouldRenderCollapseButton: PropTypes.bool.isRequired
```

Reply:
we can make it   `shouldRenderCollapseButton: PropTypes.bool.required` because even it is not the last column it will have a value (`false`)

Reply:
Fine by me 👍